assets/sass/patterns/organisms/content-header-profile.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../definitions";
+@import "../../layout/grid";
+
+.content-header-profile {
```

Comment:
We might want to add `@include wrapper(false);` (and to `content-header-simple` too) to match `content-header`?

Reply:
Good catch.